packages/pigeon/platform_tests/flutter_null_safe_unit_tests/lib/null_safe_pigeon.dart

```diff
@@ -0,0 +1,166 @@
+// Autogenerated from Pigeon (v0.1.21), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import
+// @dart = 2.12
```

Comment:
I had to add this file otherwise CI analysis steps fail.

Reply:
Maybe, we disable the analysis for `flutter_null_safe_unit_tests`? https://dart.dev/guides/language/analysis-options#the-analysis-options-file

Reply:
Thanks for the pointer.  I'm on the fence as to whether it is a good idea or not.  I think I'll keep it this way for now.